CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # [Versions](https://mui.com/versions/)
 
+## v5.15.6
+
+<!-- generated comparing v5.15.5..master -->
+
+_Jan 22, 2024_
+
+A big thanks to the 16 contributors who made this release possible. Here are some highlights ✨:
+This release was mostly about 🐛 bug fixes and 📚 documentation improvements.
+
+### `@mui/material@5.15.6`
+
+- &#8203;<!-- 29 -->[Avatar] Use variants api (#40324) @mnajdova
+- &#8203;<!-- 27 -->[material-ui][base-ui][TablePagination] Accept readonly array for `rowsPerPageOptions` prop (#40481) @pcorpet
+- &#8203;<!-- 06 -->[material-ui][PaginationItem] Deprecate classes for v6 (#40673) @sai6855
+- &#8203;<!-- 05 -->[material-ui][Rating] Fix rating width via min-content (#40503) @devhik0
+
+### `@mui/base@5.0.0-beta.33`
+
+- &#8203;<!-- 26 -->[base-ui][Select] Fix display of selected Options with rich content (#40689) @michaldudak
+- &#8203;<!-- 25 -->[base-ui][Select] Use Popup instead of Popper (#40524) @michaldudak
```

Comment:
```suggestion
- &#8203;<!-- 26 -->[Select] Fix display of selected Options with rich content (#40689) @michaldudak
- &#8203;<!-- 25 -->[Select] Use Popup instead of Popper (#40524) @michaldudak
```